src/ScottPlot4/ScottPlot/Ticks/TickCollection.cs

```diff
@@ -385,7 +390,7 @@ private string FormatLocal(double value, CultureInfo culture)
             return Math.Round(value, 10).ToString("G", culture);
         }
 
-        public (string[], string) GetPrettyTickLabels(
+        public virtual (string[], string) GetPrettyTickLabels(
```

Comment:
How come this is `virtual` now, I don't necessarily object to the change but it doesn't seem necessary given this class isn't inherited from?

Reply:
The `virtual` comes from another approach to accomplish this.  I forgot to delete it.